Fulfillment/Troubleshooting/01_HandleErrorStatus.md

```diff
@@ -0,0 +1,91 @@
+[!!Export a dispatch note](../Operation/01_ManageDispatchNotes.md#export-a-dispatch-note) 
+[!!Workflows](../../ActindoWorkFlow/Overview/01_General.md) 
+
+# Handle an error status
+
+The *Fulfillment* module manages the communication between the *Actindo Core1 Platform* and the fulfillment partner(s). Occasionally, a dispatch note or a shipment may display an error.
+
+An error status may be set for many different reasons and strongly depends on the connection and the fulfiller's own internal processes. 
+
+A dispatch note usually displays an error status when the data export from the Core1 to the partner's system has failed. A shipment, on the other hand, may display an error status when the fulfiller has reported an error, for instance because the shipment could not be arranged, but also when no feedback from fulfiller has been received. The errors are notified via the ![Notifications](../../Assets/Icons/Notifications.png) icon (Notifications) on the upper right corner as they occur.
+
+In any case, the corresponding workflow stops at the point where the error occurs. The *Actindo Core1 Platform* logging system provides detailed information on any error, which allows to identify the cause and implement the appropriate solution.
+
```

Comment:
Das ist ziemlich viel Info für ein Troubleshooting. Sicherlich alles richtig, aber nachdem ich das gelesen habe, weiß ich ehrlich gesagt noch nicht, worum es in dem Kapitel gehen soll.
Vielleicht ein bisschen kürzen? Auf jeden Fall noch prägnanter sagen, worum es hier geht.